Dockerfile

```diff
@@ -0,0 +1,118 @@
+FROM php:7.0-alpine
+
+## ENV configuration for svgo
+ENV NPM_CONFIG_LOGLEVEL info
+ENV NODE_VERSION 6.9.2
+
+RUN     apk update \
+    &&  apk upgrade \
+    &&  apk add --no-cache \
+        --repository "http://dl-4.alpinelinux.org/alpine/edge/testing" \
+        git \
+        autoconf \
+        automake \
+        build-base \
+        libtool \
+        nasm
+
+## Add PDO and MySQL
+RUN docker-php-ext-install pdo pdo_mysql
+
+## Add pngquant
+ARG INSTALL_PNGQUANT=false
+RUN if [ ${INSTALL_PNGQUANT} = true ]; then \
+    apk add --no-cache \
+    --repository "http://dl-4.alpinelinux.org/alpine/edge/testing" \
```

Comment:
Replace the URL by http://dl-4.alpinelinux.org/alpine/edge/community to enable installing pngquant again